[Kaptain] Fixing references, links and typos (#4411)
mesosphere/dcos-docs-site: pages/dkp/kaptain/2.0.0/install/air-gapped-2.2/index.md

## Add Kaptain to your Kommander Install

> The following catalog will need to be added to the Kommander install configuration file along with the DKP applications catalog:

Reviewer: Just a nit, but I think this would scan better in active voice:

```suggestion
Add the Kaptain catalog to the Kommander install configuration file along with the DKP applications catalog:
```
